solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -566,12 +577,21 @@ contract RandomBeacon is Ownable {
     ///         It removes a candidate group that was previously registered with
     ///         the DKG result submission.
     function challengeDkgResult() external {
-        // TODO: Determine parameters required for DKG result challenges.
-        dkg.challengeResult();
+        bytes32 resultHash = dkg.submittedResultHash;
+        uint256 slashingAmount = maliciousDkgResultSlashingAmount;
+        uint32[] memory maliciousMembers = dkg.challengeResult();
+        address[] memory maliciousMembersAddresses = sortitionPool
+            .getIDOperators(maliciousMembers);
 
         groups.popCandidateGroup();
 
-        // TODO: Implement slashing
+        emit DkgResultMaliciousSlashed(
+            resultHash,
+            slashingAmount,
+            maliciousMembersAddresses
+        );
+
+        staking.seize(slashingAmount, 5, msg.sender, maliciousMembersAddresses);
```

Comment:
This 5% should be governable, just like `relayEntryTimeoutNotificationRewardMultiplier` is governable. 5% in the RFC was suggested before we knew how T staking contract should look like. T staking contract does not give that 5% from `slashingAmount` just like KEEP staking contract was doing but instead, it is giving it from a special notifier reward pool.

Reply:
See https://github.com/keep-network/keep-core/pull/2695/commits/2f4e1b12c7674db2500142e70b384527b5f4afd7